### DISCUSSION TITLE
Pagination input without totals

### DISCUSSION THREAD

**Discussion by gramian** [lowdefy/lowdefy#1798] [Ideas and feature requests]
Dear Lowdefy Devs,

I would like to propose to add an option to the [Pagination Input](https://docs.lowdefy.com/Pagination) which hides the totals, so ("simple" display):
```
1-10 of 100 items < 1 / 10 >
```
would become:
```
1-10 items < 1 >
```
This would be useful in situations where it is unknown how many pages there will be in total.

Thank you for considering

PS: The `showTotal` option does not affect the part between the buttons

**Comment by Gervwyk** [comment]
Hi @gramian 
I like the suggestion, however, the Ant Design component does not support this, but building a `SimplePagination` block to support this, not based on Antd Pagination, will be possible. 

We also have this case and every time have to do the extra work to pull the total, which is most often not required.

https://4x.ant.design/components/pagination/

**Reply by gramian** [reply]
Cool,
I tried to emulate this using two `Button`s and two disabled `NumberInput`s but that led to a huge amount of logic. Getting a `SimplePagination` is very much appreciated!
